packages/client/src/StreamVideoClient.ts

```diff
@@ -271,13 +271,15 @@ export class StreamVideoClient {
    *
    * @param type the type of the call.
    * @param id the id of the call, if not provided a unique random value is used
+   * @param {boolean} [ringing] whether the call should be created in the ringing state.
    */
-  call = (type: string, id: string) => {
+  call = (type: string, id: string, ringing?: boolean) => {
```

Comment:
necessary change for creating a new call in RN when app is awake from dead through push ringing but WS would not send call.ring event